docs/how_to_add_packages.md

```diff
@@ -96,7 +97,8 @@ The system supports to use the same recipe for several versions of the library a
 
 - **N versions => 1 recipe**
 
-   Create a folder named `all` (just a convention) and put both the “conanfile.py” and the “test_package” folder there. With this approach, the “conanfile.py” won’t declare the `version` attribute.
+   Create a folder named `all` (just a convention) and put both the `conanfile.py` and the `test_package` folder there. With this approach, the `conanfile.py`
+   won’t declare the `version` attribute.
```

Comment:
```suggestion
   Create a folder named `all` (just a convention) and put both the `conanfile.py` and the `test_package` folder there.
```

We can skip that part, we never add the `version` attribute, right?

Reply:
That's an excellent question. I didn't fact check this it was already there

Reply:
https://github.com/conan-io/conan-center-index/blob/6d1c2e29d7623af9654f614c7a7d1041a5e1d1db/recipes/zlib/1.2.11/conanfile.py#L8

I don't think this pattern is used at all